src/components/PrimarySideBar.test.tsx

```diff
@@ -1,32 +1,40 @@
 import { render, screen, userEvent } from '../utils/test-utils';
 import { PrimarySideBar } from './PrimarySideBar';
 
+const emptyFn = vi.fn();
```

Comment:
`vi.fn` is for mocking. Since you don't care about this function, suggest using `_.noop` from lodash instead.

Reply:
You have a point, but I didn't wanteed to add a new dependency to the project (even though most likely we will need lodash in the future!)